src/modules/menu/components/menu/feedback/FeedbackButton.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+    <HeaderLink
+        v-if="showAsLink"
+        :primary="true"
+        data-cy="feedback-link-button"
+        @click="showFeedbackForm = true"
+    >
+        <strong>{{ $t('test_map_give_feedback') }}</strong>
+    </HeaderLink>
+    <button
+        v-else
+        class="btn btn-primary btn-sm mx-1"
+        data-cy="feedback-button"
+        @click="showFeedbackForm = true"
+    >
+        {{ $t('test_map_give_feedback') }}
+    </button>
+    <ModalWithBackdrop
+        v-if="showFeedbackForm"
+        :title="$t('feedback_rating_title')"
```

Comment:
I personally would use a short title for the window and at best the same as the feedback button. The `How would you rate your experience` would be better IMHO inside the window